app/helpers/application_helper.rb

```diff
@@ -1,5 +1,5 @@
 module ApplicationHelper
-  def full_title(page_title = '')                     
+  def full_title page_title = ''                     
```

Comment:
Use double quote instead of single quote

Reply:
Yes